modules/tensor_mechanics/include/materials/abaqus/utility/SPRINCInterface.f

```diff
@@ -0,0 +1,102 @@
+****************************************************************************************
+**  UTILITY SUBROUTINE FOR INTERFACE: SPRINC                                          **
+**  CALULATE PRINCIPAL VALUES                                                         **
+**  EQUATIONS FROM: ADVANCED STRENGTH AND APPLIED ELASTICITY: FOURTH EDITION          **
+**  ANSEL C. UGURAL, SAUL K. FENSTER PG. 505 APPENDIX B1                              **
+****************************************************************************************
+****************************************************************************************
+
+      SUBROUTINE SPRINC(S, PS, LSTR, NDI, NSHR)
+
+      INTEGER LSTR, NDI, NSHR
+      DIMENSION S(6), PS(3)
+      DOUBLE PRECISION S, I1, I2, I3, J1, J2, J3, R, T, Q, ARG, ALP,
+     1     SCALC, PRINC1, PRINC2, PRINC3
+
+      PARAMETER (ONE=1.D0,TWO=2.D0,THREE=3.D0, TWOSEVEN = 27.D0,
+     1     THIRD = ONE/THREE, TWENTYSEVENTH= ONE/TWOSEVEN,
+     2     TWOTWENTYSEVENTH = TWO/TWOSEVEN, ONETWENTYDEG=2.094395102D0,
+     3     TWOFORTYDEG=4.188790205D0)
+C
+C======================================================================+
+C-----------
+C  INPUT :
+C-----------
+C  S     	: STRESS OR STRAIN TENSOR
+C  LSTR 	: FLAG DETERMINING STRESS OR STRAIN CALCULATION
+C  NDI  	: NUMBER OF DIRECT STRESS/STRAIN COMPONENTS
+C  NSHR         : NUMBER OF SHEAR COMPONENTS
+C-----------
+C  OUTPUT :
+C-----------
+C  PS(I)	: THE THREE PRINCIPAL VALUES
+C----------------------------------------------------------------------+
+C=======================================================================
+C
+C     Calculate stress or strain invariants based on LSTR value
+
+      IF (LSTR.EQ.1) THEN
+         I1 = S(1)+S(2)+S(3)
+         I2 = (S(1)*S(2))+(S(2)*S(3))+(S(1)*S(3))
+     1        -(S(4)**2)-(S(5)**2)-(S(6)**2)
```

Comment:
Had never seen "1" to move to next line in Fortran

Reply:
Yeah, but it seems to be done consistently, with 1 for the first continuation line, 2 for the second, and so on.

Reply:
Yeah, I agree.